buildout.cfg

```diff
@@ -1,13 +1,14 @@
 [buildout]
 develop = .
-parts = interpreter test
+parts = interpreter test 
 
 [interpreter]
 recipe = zc.recipe.egg
 eggs = zope.globalrequest
-interpreter = python
+interpreter = zopepy
```

Comment:
Good reason!

(Many other zopefoundation projects use `interpreter = py` for this.  I'd never seen `zopepy` before.)

(Unrelated: I stopped using `virtualenv .` when pip started cluttering my `.` with files like `pip-selfcheck.json`.  Now I `virtualenv .env` and then sometimes symlink individual scripts bin/foo ->  .env/bin/foo.  It's annoying to do that for every script, but symlinking the entire bin/ breaks the virtualenv.)